setup.py

```diff
@@ -29,6 +29,7 @@
         "kernelci",
         "kernelci.config",
         "kernelci.lab",
+        "kernelci.data"
```

Comment:
```suggestion
        "kernelci.data",
```
to avoid making changes to this line again when appending another line.

Reply:
Fixed